cmd/crc/cmd/stop.go

```diff
@@ -54,7 +55,7 @@ func runStop(arguments []string) {
 				errors.Exit(0)
 			}
 			var userInput string
-			output.OutF("Do you want to force power off [y/n]: ")
+			output.OutF("Do you want to force power off [y/N]: ")
```

Comment:
this would have to use the `input.PromptForUserYesNo` as used in #186

Reply:
I'll rebase once #186 is in, I don't think PromptForUserYesNo exists before it.